ivy/functional/frontends/mxnet/__init__.py

```diff
@@ -1,6 +1,4 @@
-from . import numpy
 from . import numpy as np
-from . import numpy_extension
```

Comment:
These were added on purpose to allow aliases to be imported out of this namespace so best to keep them